inngest-spring-boot-demo/build.gradle.kts

```diff
@@ -26,6 +26,12 @@ dependencies {
     implementation("com.squareup.okhttp3:okhttp:4.12.0")
 
     testImplementation("org.springframework.boot:spring-boot-starter-test")
+
+    if (JavaVersion.current().isJava11Compatible) {
+        testImplementation("uk.org.webcompere:system-stubs-jupiter:2.1.6")
+    } else {
+        testImplementation("uk.org.webcompere:system-stubs-jupiter:1.2.1")
+    }
```

Comment:
I don't like this at all but it works and it's a test dependency. https://github.com/webcompere/system-stubs?tab=readme-ov-file#system-stubs

> **⚠ WARNING: JDK Support.**
> _This project has now moved to a JDK11 minimum version_
>
> The v2.x branch is the LTS version. However, there is best effort support to keep the Java 8 compatible
> [v1.x](https://github.com/webcompere/system-stubs/tree/1.x) branch.

Reply:
I see, that sucks we can't just use one version for everything but this seems like an OK solution given they've moved their LTS to a newer JDK than inngest-kt's minimum